openmeetings-web/src/main/java/org/apache/openmeetings/web/common/tree/FileTreePanel.java

```diff
@@ -158,8 +158,7 @@ protected AbstractLink newBaseButton(String markupId, IModel<String> labelModel,
 
 		public void onDownlownClick(AjaxRequestTarget target, String ext) {
 			BaseFileItem fi = getLastSelected();
-			File f = ext == null && (Type.IMAGE == fi.getType() || Type.PRESENTATION == fi.getType())
-					? fi.getOriginal() : fi.getFile(ext);
```

Comment:
In the original code
In case `ext == null` the method `BaseFileItem.getOriginal()` will do the job